src/Raven.Client/Documents/Commands/Batches/PutAttachmentCommandData.cs

```diff
@@ -9,7 +9,7 @@ namespace Raven.Client.Documents.Commands.Batches
 {
     public class PutAttachmentCommandData : ICommandData
     {
-        public PutAttachmentCommandData(string documentId, string name, Stream stream, string contentType, string changeVector)
+        public PutAttachmentCommandData(string documentId, string name, Stream stream, string contentType, string changeVector, bool fromEtl = false)
```

Comment:
This is a breaking change. You need to create 2nd ctor (internal) and call it from this one

Reply:
Ok